kirc.c

```diff
@@ -249,18 +248,18 @@ static void refresh_line(state l)
         txtlenb += u8_next(buf + txtlenb, 0);
     ab_initialize(&ab);
     snprintf(seq, sizeof(seq), "\r");
-    ab_append(&ab, seq, strnlen(seq, MSG_MAX));
+    ab_append(&ab, seq, strnlen(seq, 64));
```

Comment:
why `64` for maxlen?

Reply:
Because seq was only defined so big.
This is how it was defined before I made any changes.
Making the number any higher only introduced possible overflows.
Another solution would have been defining seq larger, but I decided to not increase it's size as to preserve original functionality.
If you want, I can increase all sizes to MSG_MAX, but it made no sense to define seq with length 64 and treat it as if it was of length MSG_MAX.

Reply:
@mcpcpc @emekoi 
So, is everything all right? Or should we use the second approach?

Reply:
> @mcpcpc @emekoi So, is everything all right, or should we use the second approach?

leaving it as-is should be fine.